pkg/router/template/router_test.go

```diff
@@ -391,11 +461,14 @@ func TestCreateServiceAliasConfig(t *testing.T) {
 	expectedSUs := map[ServiceUnitKey]int32{
 		suName: serviceWeight,
 	}
+	httpResponseHeadersList := []HTTPHeader{{Name: "X-Frame-Options", Value: "'DENY'", Action: "Set"}, {Name: "X-XSS-Protection", Value: "'1;mode=block'", Action: "Set"},
+		{Name: "X-Frame-Options", Action: "Delete"}, {Name: "X-XSS-Protection", Action: "Delete"}}
+	httpRequestHeadersList := []HTTPHeader{{Name: "Accept", Value: "'text/plain,text/html'", Action: "Set"}, {Name: "x-client", Value: `'"abc"\ '\''def'\'''`, Action: "Set"}, {Name: "Accept-Encoding", Action: "Delete"}}
 
 	// Basic sanity, validate more fields as necessary
 	if config.Host != route.Spec.Host || config.Path != route.Spec.Path || !compareTLS(route, config, t) ||
 		config.PreferPort != route.Spec.Port.TargetPort.String() || !reflect.DeepEqual(expectedSUs, config.ServiceUnits) ||
-		config.ActiveServiceUnits != 0 {
+		!cmp.Equal(config.HTTPResponseHeaders, httpResponseHeadersList) || !cmp.Equal(config.HTTPRequestHeaders, httpRequestHeadersList) || config.ActiveServiceUnits != 0 {
```

Comment:
This line is rather long.  If you format the lines this way, you make both the diff and the patched code easier to read:
```suggestion
		config.ActiveServiceUnits != 0 ||
		!cmp.Equal(config.HTTPResponseHeaders, httpResponseHeadersList) ||
		!cmp.Equal(config.HTTPRequestHeaders, httpRequestHeadersList) {
```

Reply:
ok